### DISCUSSION TITLE
阿里云盘现在分为备份盘和资源库，资源库怎么挂载

### DISCUSSION THREAD

**Discussion by mrmcree** [AlistGo/alist#4918] [Q&A]
默认挂载的root路径好像是备份盘，资源库怎么挂载
![image](https://github.com/alist-org/alist/assets/22073894/dc6e83d5-f5c6-44a0-aae9-908b231a69f3)

**Comment by xhofe** [comment]
将在下个版本支持资源盘。

**Reply by jesustoachild** [reply]
更新至 v3.25.1 版本后，还是没办法访问资源盘？

**Reply by chenshuai19960520** [reply]
云盘类型选择了资源库 但是还是显示备份盘内容

**Reply by anwen-anyi** [reply]
> 云盘类型选择了资源库 但是还是显示备份盘内容

如果是在原来的基础上修改的，保存好进入列表右下角刷新一下
